### DISCUSSION TITLE
How do I subscribe to an issue in GitHub for Mobile?

### DISCUSSION THREAD

**Discussion by tddschn** [community/community#3670] [Mobile] [answered]
I was viewing an issue on the mobile app and only found a button to comment on the issue - how do I subscribe to an issue in GitHub for Mobile?

**Comment by brianlovin** [comment, marked as answer]
Hey @tddschn:

- On iOS, if you swipe the sheet up from the bottom of the issue view (the sheet that the Comment button is on), the subscription options are at the bottom of the sheet.
- On Android, the notification subscription button is in the top navigation bar of the issue view

![IMG_8C620CB39AD3-1](https://user-images.githubusercontent.com/1923260/113741829-e24f6380-96b6-11eb-87c7-e3c56cd8e369.jpeg)

**Reply by ayvazjet** [reply]
> #- [ ] - **_[]()_**@~~````

````~~Hülya
